content/reference/api/http.md

```diff
@@ -54,29 +66,33 @@ Alternatively, you can use basic authentication in the request header following
 
-OpenBazaar is also a protocol for buying and selling any good or service with cryptocurrency. Using Ricardian contracts and multisignature escrow, buyers and sellers can transact safely with mechanisms to recover funds in the event of a dispute or accident.
+*(Note: The API (and underlying protocol) is under development and we reserve the right to shift the API until it becomes more mature. Despite this, the core usage of the API changes very little with most of the protocol shifting hidden behind it. It is not recommended attempting to access or reproduce the protocol logic or use internals until a formal protocol library is published for external use.)*
 
-When the OpenBazaar server starts for the first time, it creates a public directory of data that is hosted or 'seeded' by other peers:
+To develop applications on top of platform, the APIs must be understood in context of the application architecture and the OpenBazaar protocol. A summary of this architecture would be to describe it as using IPFS as a layer of persistence for content which describes offers for trade (in the form of Listings) and as well as a means of transporting messages (which are peer-to-peer and encrypted) to achieve commerce around these Listings. The OpenBazaar protocol is behavior built on top of this persistence and communication layer which uses Ricardian Contracts and cryptocurrencies to enable trustless and permissionless trade.
 
+When the OpenBazaar server starts for the first time, it generates a unique identity (called your Peer ID) and a public directory of data that is hosted or 'seeded' by other peers. This identity and public data is distributed into the live network and is what allows your data to be available to others on-demand. By default, the public data includes boilerplate profile with random data associated to your Peer ID and an empty store without Listings. Depending on how you connect to the internet, your public data would also include any IP addresses which originated the data and remains in the network routing tables until they expire.
 
+The first step after generating an identity is to create a public profile and configure your private settings for the node. Creating a public facing profile will allow other users to see details such as display name, avatar, and header image when they visit your node. This can be done with `PUT /ob/profile`. The Peer ID can be retrieved by calling `GET /ob/config`. The settings allow you to set your preferred currency (to display prices), country, create a delivery etc; these setting are private and are not visible to the network. This can be done with `POST /ob/settings` or `PUT /ob/settings`.
```

Comment:
I believe the `PUT /ob/profile` can only be called to update the profile. `POST /ob/profile` would be used to create it.